cloud-provider.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 AZURE_META_URL="http://169.254.169.254/metadata/instance/compute"
+AZURE_META_API_VERSION="2019-11-01"
```

Comment:
```
The version 2019-11-01 is currently getting deployed and may not be available in all regions.
```

Is this going to bite us? https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service

Reply:
Maybe we should use 2019-08-15 as that's in all regions?

Reply:
ya taht's fine, also why i made it a param

Reply:
Let's default to `2019-08-15`.

Reply:
2019-08-15 it is